### PR TITLE
fmui export: 392 enable doil to start export by cron

### PR DESCRIPTION
**Pull request by daniwe4**
Der mysql Part wurde wieder mit reingenommen. Beim import wird der doch benötigt.
Https ist per default erstmal aus.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,13 @@ build with doil (instances from version >=1.1) are able to be exported.
 
 See `doil pack:<command> --help` for more information
 
+To start the command `doil pack:import` with cron you should add the parameter -T or --no-term to the call. Otherwise you will get an error message that no TTY can be found.
```
Comment (tbongers-cat) [inline README.md:202]: Why does this refer to `pack:import`, while the following example is for `pack:export`? And what is the difference between the two parameters? We should explain this better.

File: app/src/Commands/Pack/ExportCommand.php [hunk 2]
```diff
@@ -44,6 +44,7 @@ public function configure() : void
         $this
             ->addArgument("instance", InputArgument::REQUIRED, "Name of the instance to export")
             ->addOption("global", "g", InputOption::VALUE_NONE, "Determines if the instance is global")
+            ->addOption("cron", "c", InputOption::VALUE_NONE, "Must be use for triggering export by cron")
```
Comment (tbongers-cat) [inline app/src/Commands/Pack/ExportCommand.php:47]: Must be **used** for triggering **an** export ...

File: setup/doil.sh [hunk 3]
```diff
@@ -19,12 +19,23 @@ doil_get_conf() {
   echo ${VALUE}
 }
 
+# If -T or --no-term is selectet, the docker command runs without
```
Comment (tbongers-cat) [inline setup/doil.sh:22]: **selected**

**Review by tbongers-cat — CHANGES_REQUESTED** [review]
Codewise this looks good, but the explanation needs some work imho.
Do we really need two parameters for this? And if so, the documentation should make that more obvious.

File: app/src/Commands/Pack/ExportCommand.php (hunk 2 above)
Comment (daniwe4) [reply app/src/Commands/Pack/ExportCommand.php:47]: Done!

File: setup/doil.sh (hunk 3 above)
Comment (daniwe4) [reply setup/doil.sh:22]: Done!

File: README.md (hunk 1 above)
Comment (daniwe4) [reply README.md:202]: Done!

**Review by tbongers-cat — APPROVED** [review]
Much better documentation now, thanks =)

**Pull request merged**